types/meteor/accounts-base.d.ts

```diff
@@ -54,15 +57,29 @@ declare module "meteor/accounts-base" {
     }
 
     module Accounts {
-        function changePassword(oldPassword: string, newPassword: string, callback?: (error?: Error | Meteor.Error | Meteor.TypedError) => void): void;
-
-        function forgotPassword(options: {
-            email?: string;
-        }, callback?: (error?: Error | Meteor.Error | Meteor.TypedError) => void): void;
-
-        function resetPassword(token: string, newPassword: string, callback?: (error?: Error | Meteor.Error | Meteor.TypedError) => void): void;
-
-        function verifyEmail(token: string, callback?: (error?: Error | Meteor.Error | Meteor.TypedError) => void): void;
+        function changePassword(
+            oldPassword: string,
+            newPassword: string,
+            callback?: (error?: Error | Meteor.Error | Meteor.TypedError) => void,
+        ): void;
+
+        function forgotPassword(
+            options: {
+                email?: string;
+            },
```

Comment:
In a couple of places, the formatting expands small and simple objects like this one into multiple ones, just because it was a multi-line object before. How about wrapping them back into one line? It'd shave off a couple of lines from the files and make them _slightly_ more comprehensible.